packages/nimble/R/nimbleFunction_keywordProcessing.R

```diff
@@ -846,8 +799,7 @@ keywordList[['nimCopy']] <- nimCopy_keywordInfo
 keywordList[['[[']] <- doubleBracket_keywordInfo
 keywordList[['$']] <- dollarSign_keywordInfo
 keywordList[['[']] <- singleBracket_keywordInfo
-keywordList[['nimEigen']] <- nimbleListReturningFunction_keywordInfo
```

Comment:
Can you explain why `nimEigen` "just works" now, even without needing `nimbleListReturningFunction_keywordInfo`? I ask because the `nimOptim` handling below is obsolete and unused, and will be removed in #336